src/chains/juno.ts

```diff
@@ -1,10 +1,33 @@
-import { CosmWasmClient } from "@cosmjs/cosmwasm-stargate";
 import { GetOwnedNftImageUrlFunction } from "../types";
-import { KnownError, NotOwnerError } from "../error";
+import { KnownError } from "../error";
 import { secp256k1PublicKeyToBech32Address } from "../utils";
-import * as Cw721 from "../cw721";
+import { ApolloClient, InMemoryCache, gql } from "@apollo/client";
+import { getOwnedNftImageUrl as makeCw721GetOwnedNftImageUrl } from "./cw721";
 
 const JUNO_RPC = "https://rpc.juno.strange.love:443";
+const LOOP_API_TEMPLATE = "https://nft-juno-backend.loop.markets";
+
+const GET_LOOP_NFTS_QUERY = gql`
```

Comment:
What does a `gql`? Graphical query language (??). A comment could be helpful here to new contributors.

Reply:
Graph Query Language :) Sure!